examples/now2/next.config.js

```diff
@@ -5,6 +5,9 @@ const withOffline = moduleExists('next-offline')
 const nextConfig = {
   target: "serverless",
   workboxOpts: {
+    // Trying to set NODE_ENV=production when running yarn dev causes a build-time error so we
```

Comment:
@hanford presumably I should revert this before merging? It would be nice if these examples had some "prod-like" command so that you can test locally but with the real deal SW.